src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -355,23 +355,22 @@ private static void setInstanceEdits(int personId) throws Exception
 	private static Map<String, List<String>> downloadAndProcessWormbaseFile(URL wormbaseUrl) throws IOException {
 		File wormbaseFile = Paths.get(wormbaseUrl.toString()).getFileName().toFile();
 		Map<String, List<String>> wormbaseMappings = new HashMap<>();
-		if (!wormbaseFile.exists()) {
-			FileUtils.copyURLToFile(wormbaseUrl, wormbaseFile);
-		}
+		FileUtils.copyURLToFile(wormbaseUrl, wormbaseFile);
 		BufferedReader br = new BufferedReader(new InputStreamReader(new GZIPInputStream(new FileInputStream(wormbaseFile))));
 		String line = null;
 		while ((line = br.readLine()) != null) {
 			String wormbaseGeneId = line.split(",")[1];
 			String wormbaseGeneName = line.split(",")[2];
 			if (!wormbaseGeneId.isEmpty() && !wormbaseGeneName.isEmpty()) {
-				if (wormbaseMappings.get(wormbaseGeneId) != null) {
+				if (wormbaseMappings.containsKey(wormbaseGeneId)) {
 					wormbaseMappings.get(wormbaseGeneId).add(wormbaseGeneName);
 				} else {
-					wormbaseMappings.put(wormbaseGeneId, new ArrayList<>(Arrays.asList(wormbaseGeneName)));
+					wormbaseMappings.computeIfAbsent(wormbaseGeneId, k -> new ArrayList<>()).add(wormbaseGeneName);
 				}
 			}
 		}
 		br.close();
+		wormbaseFile.delete();
```

Comment:
Totally optional, but you could do `Files.delete(wormbaseFile)` if you'd like to use the java.nio.Files API.